src/main/java/com/github/fishio/PlayingField.java

```diff
@@ -316,7 +324,7 @@ public void preListeners(boolean render) {
 				tl.preTick();
 			} catch (Exception ex) {
 				//TODO Handle exception differently
-				System.err.println("Error in preTick!");
+				log.log(LogLevel.ERROR, "Error in preTick!");
 				ex.printStackTrace();
```

Comment:
Correct me if I'm wrong, but instead of ex.printStackTrace(); can't you do log.log(LogLevel.ERROR, "Error in preTick!\t" + ex.getMessage() )